packages/frontend-defaults/report.api.md

```diff
@@ -53,7 +53,7 @@ export interface CreateAppOptions {
   pluginInfoResolver?: FrontendPluginInfoResolver;
 }
 
-// @public
+// @public @deprecated (undocumented)
```

Comment:
Could we delete it immediately since we warned that it is a breaking change in the changeset?

Reply:
Opted for removing the breaking notice and keep it around as deprecated for now. Figured there's no significant benefit to removing it straight away.